src/i_sound.c

```diff
@@ -549,13 +549,17 @@ void I_InitSound(void)
     ALCdevice *device;
     ALCcontext *context;
 
-    if (nosfxparm)
+    if (nosfxparm && nomusicparm)
     {
         return;
     }
 
     printf("I_InitSound: ");
 
+#ifndef _WIN32
```

Comment:
Sorry, I think we should rather use `WOOFDATADIR` here.

Reply:
Done in https://github.com/fabiangreffrath/woof/pull/981/commits/d9945372125dce3df29baac2d4a681379a71aa4f